src/Thoth.Json.Net/Encode.fs

```diff
@@ -186,6 +186,9 @@ module Encode =
     let datetimeOffset (value : System.DateTimeOffset) : JsonValue =
         JValue(value.ToString("O", CultureInfo.InvariantCulture)) :> JsonValue
 
+    let timespan (value : System.TimeSpan) : JsonValue =
+        JValue(value.ToString()) :> JsonValue
```

Comment:
Does .Net support `CultureInfo.InvariantCulture` for timespan ? If yes, we need to had it.

Reply:
Alfonso also asked for this. I will check with him.